packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -748,6 +769,38 @@ const TaxIdentificationNumberField = ({ disabled, required = false }) => {
                 disabled={disabled}
                 required={required}
             />
+            <div
+                data-testid='tax_identification_number_pop_over'
+                onClick={e => {
+                    setIsTaxResidencePopoverOpen(false);
+                    setIsTinPopoverOpen(true);
+                    if (e.target.tagName !== 'A') e.stopPropagation();
+                }}
+            >
+                <Popover
+                    alignment={isDesktop ? 'right' : 'left'}
+                    icon='info'
+                    is_open={is_tin_popover_open}
+                    message={
+                        <Localize
+                            i18n_default_text={
+                                "Don't know your tax identification number? Click <0>here</0> to learn more."
+                            }
+                            components={[
+                                <a
+                                    key={0}
+                                    className='link link--red'
+                                    rel='noopener noreferrer'
+                                    target='_blank'
+                                    href='https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/'
```

Comment:
cant we use OECD_TIN_FORMAT_URL from account/src/Constants/external-urls.ts

Reply:
Done 🙇